File: reto3/fibonacci.py
def fibonacci(n):
    num1 = 0
    num2 = 1
    result = [0, 1, ]
    
    for i in range(n):
        result.append(num1+num2)
        aux = num1 + num2
        num1 = num2
        num2 = aux
    return print(result)

def run():
    limit = int(input("How many digit do you want?: "))
    fibonacci(limit)

if __name__=='__main__':
    run()

File: reto3/Fibonacci.java
public class Fibonacci{

    public static void main(String[] args) {
        
        int num1 = 0, num2 = 1;
        int [] Fibonacci = new int[20];
        System.out.print(num1+", "+num2+ ", ");
        for (int i = 0; i < 20; i++) {
            
            int result = num1 + num2;
            Fibonacci[i] = num1 + num2;
            num1 = num2;
            num2 = result;
            System.out.print(Fibonacci[i]+", ");
        
        
        }

    }
}